build.proj

```diff
@@ -5,7 +5,7 @@
     <IsTraversalBuild>true</IsTraversalBuild>
   </PropertyGroup>
   
-  <Import Project="dir.props" />
+  <Import Project="src\dir.props" />
```

Comment:
Well, the idea is that when we are building projects from src, we need to process the .sln file, while projects in Samples have no solution. So these properties do belong in the src\dir.props.